packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1481,12 +1471,12 @@ This operation:
 * Supports [pagination](/docs/reference/core-okta-api/#pagination).
 * Requires [URL encoding](http://en.wikipedia.org/wiki/Percent-encoding).
 For example, `search=profile.department eq "Engineering"` is encoded as `search=profile.department%20eq%20%22Engineering%22`.
-Examples use cURL-style escaping instead of URL encoding to make them easier to read.
 Use an ID lookup for records that you update to ensure your results contain the latest data.
+* Special characters included in a search must also use URL encoding. For example, searching for `Smith"`, must be escaped, `Smith\"`, and then encoded: `Smith%5c%22`. The full encoded search appears as: `api/v1/users?search=profile.lastName%20eq%20%22Smith%5C%22%22`.
```

Comment:
Hi Chenning, thanks for the great feedback:

Previous comment: Yes, I have several tasks created to review curl examples in other parts of the Reference ([OKTA-468550](https://oktainc.atlassian.net/browse/OKTA-468550)). A couple of customer tickets have come in on this; I'm slowly working through my list.

1. I was following the format from the bullet [above](https://developer.okta.com/docs/reference/api/users/#list-users-with-search) the one I added. I'm not sure if they're interchangeable, although I can test ("+" worked in my curl examples, but is encoding necessary if referenced in code?)
2. Thanks for your updated example, which makes more sense. I'll update the PR with this info.
3. Quick question on the `search` format or syntax. Search worked in curl `/api/v1/users?search=profile.mobilePhone+sw+%22555%22+and+status+eq+%22ACTIVE%22"` but I couldn't get that to work in Postman (the `filter` calls worked in both Postman and curl).

Reply:
Re: (2) ... the ticket I'm referencing [OKTA-461913](https://oktainc.atlassian.net/browse/OKTA-461913) implies that the escape character ALSO needs encoding, as follows: `/api/v1/users?search=profile.login+eq+%22test.admin%5c%40okta.com%22` Can you confirm? I'll see if I can test this locally.

Reply:
> Quick question on the search format or syntax. Search worked in curl /api/v1/users?search=profile.mobilePhone+sw+%22555%22+and+status+eq+%22ACTIVE%22" but I couldn't get that to work in Postman (the filter calls worked in both Postman and curl).

You seem to have an extra quotation mark at the end of your search(?). It worked for me in Postman after removing it. ie. `/api/v1/users?search=profile.mobilePhone+sw+%22555%22+and+status+eq+%22ACTIVE%22` worked for me in Postman.

> Re: (2) ... the ticket I'm referencing [OKTA-461913](https://oktainc.atlassian.net/browse/OKTA-461913) implies that the escape character ALSO needs encoding, as follows: /api/v1/users?search=profile.login+eq+%22test.admin%5c%40okta.com%22 Can you confirm? I'll see if I can test this locally.

The `@` in user login `test.admin@okta.com` doesn't need to be escaped. ie. pre encoding, the request should look like `/api/v1/users?search=profile.login eq "test.admin@okta.com"`, instead of `/api/v1/users?search=profile.login eq "test.admin\@okta.com"`.

Reply:
@chenningzhang-okta confirmed from my simple testing that special characters don't need to be escaped within quoted text (although quotes require url encoding), e.g., both `/api/v1/users?search=profile.login+eq+%22test.admin@okta.com%22` and `/api/v1/users?search=profile.login+eq+%22test.admin%40okta.com%22` work as curl calls.

Pinged original bug creator for more details.